api/api.proto

```diff
@@ -951,6 +951,74 @@ service OpenStorageCluster {
     }
 }
 
+// OpenStorageClusterPair service provides the methods to manage a cluster pair
+service OpenStorageClusterPair{
+  // Creates Pair with a remote cluster and returns details about the remote cluster
+  //
+  // ##### Example
+  // {% codetabs name="Golang", type="go" -%}
+  // id, err := client.Create(context.Background(), &api.ClusterPairCreateRequest{
+  //   RemoteClusterIp: "127.0.0.1",
+  //   RemoteClusterPort: 12345,
+  //   RemoteClusterToken: "<Auth-Token>",
+  //   SetDefault: <Boolean-Value>,
+  // })
+  // {%- endcodetabs %}
+  rpc Create(ClusterPairCreateRequest)
+    returns (ClusterPairCreateResponse){
+      option(google.api.http) = {
+        post: "/v1/clusterspairs"
+        body: "*"
+      };
+    }
+    // ProcessRequest handles a remote cluster's pair request
+    //
+    // ##### Example
+    // {% codetabs name="Golang", type="go" -%}
+    // id, err := client.Create(context.Background(), &api.ClusterPairProcessRequest{
+    //   SourceClusterId: "ID",
+    //   RemoteClusterToken: "<Auth-Token>",
+    // })
+    // {%- endcodetabs %}
+    rpc ProcessRequest(ClusterPairProcessRequest)
+      returns (ClusterPairProcessResponse){
+        option(google.api.http) = {
+          post: "/v1/clusterspairs/process"
+          body:"*"
+        };
+      }
+      // Get information about a cluster pair
+      rpc Get(ClusterPairGetRequest)
+        returns (ClusterPairGetResponse){
+          option(google.api.http) = {
+            get: "/v1/clusterspairs/{id}"
+          };
+        }
+      // Enumerate returns list of cluster pairs
+      rpc Enumerate(SdkClusterPairsEnumerateRequest)
+        returns (ClusterPairsEnumerateResponse){
+          option(google.api.http) = {
+            get: "/v1/clusterspairs"
+          };
+        }
+        // GetToken returns a auth token
+        rpc GetToken(ClusterPairTokenGetRequest)
```

Comment:
ClusterPairTokenGetRequest takes a value as a command, so this cannot be a `Get` call.

Let's chage this to just `Token()` and be a `post` on `/v1/clusterpairs/token` REST.

Reply:
Post shouldn't be used for read operarions